codegen.ts

```diff
@@ -27,15 +27,15 @@ const gqlConfig: CodegenConfig = {
 		'./src/__generated__/gateway/': {
 			documents: ['src/**/gql/gateway/*.ts'],
 			preset: 'client',
-			schema: `${process.env.NEXT_PUBLIC_GATEWAY_API_ROOT}/graphql`,
+			schema: `${process.env.NEXT_PUBLIC_GATEWAY_API_ROOT}`,
 			presetConfig: {
 				gqlTagName: 'gql',
 			},
 		},
 		'./src/__generated__/clinical/': {
 			documents: ['src/**/gql/clinical/*.ts'],
 			preset: 'client',
-			schema: `${process.env.NEXT_PUBLIC_CLINICAL_API_ROOT}/graphql`,
+			schema: `${process.env.NEXT_PUBLIC_CLINICAL_API_ROOT}`,
```

Comment:
new rdpc work uses `/gql` instead of platform `/graphql`
remove hardcoding and use full url from env var